designs/index.md

```diff
@@ -7,25 +7,248 @@ title: Design Documents
+
+
+## I'm a reviewer. What are my responsibilities?
+
+
+### When you receive a new proposal
+
+*   Take a quick look at the document. Comment if critical information is
+    missing, or if the design doesn't fit with the goals of the project.
+*   Suggest additional reviewers.
+*   Approve the PR when it is ready for review.
+
+
+### During the review process
+
+*   Engage in a dialogue with the design author about issues that are
+    problematic or require clarification.
+*   If appropriate, invite comments from non-reviewers who should be aware of
+    the design.
+*   Decide which comments must be addressed by the author as a prerequisite to approval.
+*   Write "LGTM" (_Looks Good To Me_) in the discussion thread when you are
+    happy with the current state of the proposal.
+
+If you get a design review request, please make sure it followed this process.
+Do not approve designs affecting Bazel if they are not in the [design
+index](https://github.com/bazelbuild/proposals).
+
+
+## I'm a lead reviewer. Do I have any additional responsibilities?
+
+You're ultimately responsible for making a go/no-go decision on implementation
+of a pending design. If you're not able to do this, you should identify a
+suitable delegate (reassign the PR to the delegate), or reassign the bug to a
+Blaze manager for further disposition.
+
+
+### During the review process
+
+*   Ensure that the comment and design iteration process moves forward
+    constructively.
+*   Prior to approval, ensure that concerns from other reviewers have been
+    resolved.
+
+
+### Once the design has been approved by all reviewers
+
+*   Make sure there has been at least 1 week since the announcement on a
+    mailing-list.
+*   Make sure the PR updates the status.
+*   Approve the PR sent by the proposal author.
+
+
+### To reject the design
+
+*   Make sure the PR author sends a PR; or send them a PR.
+*   The PR updates the status of the document.
+*   Add a comment to the document explaining why the design can't be approved in
+    its current state, and outlining next steps, if any (e.g., "revisit invalid
+    assumptions and resubmit").
+
+
+## What is a significant change?
+
+
+There are no hard and fast criteria, but here are some examples:
+
+*   Addition or deletion of native build rules
+*   Breaking-changes to native rules
+*   Changes to a native build rule semantics that affect the behavior of more
+    than a single rule
+*   Changes to Bazel's rule definition API
+*   Changes to the APIs that Bazel uses to connect to other systems
+*   Changes to the Skylark language, semantics, or APIs
+*   Changes that could have a pervasive effect on Bazel performance or memory
+    usage (for better or for worse)
+*   Changes to widely used internal APIs
+*   Changes to flags and command-line interface.
+
+When a proposal adds, removes, or modifies any function or object available in
+BUILD, WORKSPACE, or bzl files, Skylark team has to be in the reviewers list.
```

Comment:
Should you clarify that the intent is modifies ... available in *all* bzl files, rather than
adds a function to any specific .bzl file(s).